kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/util/DynamicBean.kt

```diff
@@ -24,25 +28,25 @@ internal class DynamicBean(val origin: Any) {
     }
 
     @Suppress("UNCHECKED_CAST")
-    fun <T> property(name: String): T {
+    fun <T> value(name: String): T {
         return gradleWrapper.getProperty(name) as T
```

Comment:
For debuggability reasons (i.e. to get more meaningful exception than `gradleWrapper.getProperty(name) must not be null` you can add `is` check and then throw something with `name` and `gradleWrapper.toString()` in it.